internal/engine/upper.go

```diff
@@ -516,6 +518,10 @@ func handleFSEvent(
 	}
 }
 
+func handleGitBranchStatus(state *store.EngineState, bsa fswatch.GitBranchStatusAction) {
+	// TODO(nick): Do something with this data.
```

Comment:
do you have a plan for how to handle it if we get "aah you changed the branch" signals from all 10 of our resources because they all reside in the same git repo?

Reply:
ah whoops, i misread a bit of the code, never mind!